src/components/AlertMsg/index.jsx

```diff
@@ -0,0 +1,37 @@
+import React, { useState, Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { Alert } from 'react-bootstrap';
+import './alert.scss';
+
+const AlertMsg = (props) => {
```

Comment:
@chingsley I thought you and @olajide1234 agreed on using just one alert component?